pkg/maps/lbmap/maglev.go

```diff
@@ -201,7 +202,7 @@ func (k *MaglevInnerKey) String() string            { return fmt.Sprintf("%d", k
 // +k8s:deepcopy-gen=true
 // +k8s:deepcopy-gen:interfaces=github.com/cilium/cilium/pkg/bpf.MapValue
 type MaglevInnerVal struct {
-	BackendIDs []uint16
+	BackendIDs []uint32
```

Comment:
This should be `loadbalancer.BackendID`, no?

Reply:
It won't regenerate correct deepcopy method if it is `loadbalancer.BackendID` :)